src/main/java/hudson/plugins/sauce_ondemand/SauceOnDemandBuildWrapper.java

```diff
@@ -855,6 +862,20 @@ public SauceConnectHandler(
             this.verboseLogging = sauceOnDemandBuildWrapper.isVerboseLogging();
             this.sauceConnectPath = sauceOnDemandBuildWrapper.getSauceConnectPath();
             this.sauceConnectJar = sauceConnectJar;
+            try {
+                this.maxRetries = Integer.parseInt(maxRetries);
+            } catch (NumberFormatException e) {
```

Comment:
No, it will not, I think this is already caught by the `NumberFormatException`. 
I also tried it in a separate piece of code and a NPE is not raised. Here is the snippet:
```
    String value = null;
    int maxRetries;
    try {
      maxRetries = Integer.parseInt(value);
    } catch (NumberFormatException e) {
      maxRetries = 10;
    }
    System.out.println(maxRetries);
```
It prints 10 at the end.
I also tried it when the plugin has no configuration at all, then the values are null, and the try-catch handled it.